test/test.js

```diff
@@ -93,6 +85,14 @@ test('db.row (template string)', async function (t) {
   )
 })
 
+test('db.row (template string with no args)', async function (t) {
```

Comment:
First version introduced a regression with the form
```js
await db.value`query`
```

Added this test to make sure it doesn't regress again.